File: user-service/src/main/java/com/study/user/UserApplication.java
package com.study.user;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {"com.study.user", "com.study.common"})
public class UserApplication {

    public static void main(String[] args) {
        SpringApplication.run(UserApplication.class, args);
    }
}

File: user-service/src/main/kotlin/com/study/user/UserApplication.kt
package com.study.user

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.study.user", "com.study.common"])
class UserApplication

fun main(args: Array<String>) {
    runApplication<UserApplication>(*args)
}
